src/native/public/mono/jit/mono-private-unstable.h

```diff
@@ -0,0 +1,25 @@
+/**
+ * \file
+ *
+ * Private unstable APIs.
+ *
+ * WARNING: The declarations and behavior of functions in this header are NOT STABLE and can be modified or removed at
```

Comment:
nit: it would be nice to retain this warning in mono-private-unstable-types.h and mono-private-unstable-functions.h too.